### DISCUSSION TITLE
SIMD support in WASM？

### DISCUSSION THREAD

**Discussion by troy351** [swc-project/swc#2169] [Ideas]
Since rust 1.54.0 and Chrome 91 supports SIMD now

**Comment by kdy1** [comment]
It will break @swc/wasm for lots of people

**Reply by troy351** [reply]
Maybe next major release？

**Reply by kdy1** [reply]
What do you mean?
swc@v2 does not have to support users who does not use chrome?

**Reply by troy351** [reply]
Sorry for missing the downgrade part. It's too early to discuss this.

**Reply by troy351** [reply]
I found Firefox 90 and Node.js 16.4 also supports SIMD, so the problem is Safari.

https://github.com/WebAssembly/simd/blob/main/proposals/simd/ImplementationStatus.md

https://webassembly.org/roadmap/

**Reply by kdy1** [reply]
Some users have cpu without `aes-ni`. I think enabling `sse2` can be ok in future, but I'm not sure about more than it.

**Reply by troy351** [reply]
I'm no expert of this either.
Just giving this advice
